hippo4j-spring-boot/hippo4j-spring-boot-starter/src/main/java/cn/hippo4j/springboot/starter/core/ClientWorker.java

```diff
@@ -147,7 +138,7 @@ public void run() {
                 String itemId = keys[1];
                 String namespace = keys[2];
                 try {
-                    String content = getServerConfig(namespace, itemId, tpId, 3000L);
+                    String content = getServerConfig(namespace, itemId, tpId, HTTP_EXECUTE_TIMEOUT);
```

Comment:
Why did 3000 become 5000?

Reply:
because the introduced HTTP timeout setting is a global constant in hippo4j-common, And I have now reverted it to 3000 by creating a constant in the current file.